ansible/README.md

```diff
@@ -1,10 +1,42 @@
 # Ansible Playbooks
 
+## Database Initialization
+
+The following databases are created by the `postgresql/init.yml` playbook:
+
+| Database      | Owner   |
+| ------------- | ------- |
+| de            | de      |
+| notifications | de      |
+| metadata      | de      |
+| de_releases   | de      |
+| grouper       | grouper |
+| qms           | de      |
+| unleash       | de      |
+| k3s           | de      |
+
+The owner users are configurable through the `dbms_connection_user` and `grouper_connection_user` group_vars.
+
+Migrations are run for the `de`, `metadata`, `notifications`, and `de_releases` databases. The `k3s` database is initialized by the installation process for the k3s cluster and the `grouper` database is handled by it's own playbook since it's fairly complicated. The `qms`
+database is created here, but populated by the `qms` service. `unleash` is not yet initialized by this playbook.
+
+**NOTE** The `-e "@</path/to/dbms/group_vars>` setting is required here because the `hosts` setting in the playbook is localhost and ansible won't pick up the vars in the dbms group. If you have a localhost group_vars file then that won't be necessary to include.
+
+| Playbook            | Description                                    | Example                                                                                   |
+| ------------------- | ---------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| postgresql/init.yml | Creates the databases and runs some migrations | `ansible-playbook -i <inventory> -e "@</path/to/dbms/group-vars>" -K postgresql/init.yml` |
```

Comment:
Would it be possible to set `hosts` to the DBMS host and use `delegate_to` to have the commands run on the local host? That way, we might be able to eliminate the need to load the group vars separately. I don't know if that will work or not, but I thought I'd ask. 😁

Reply:
I can give that a shot. I seem to remember trying that out, but I tried out a bunch of stuff and may be misremembering things.

Reply:
Turns out I didn't try it out and it worked fine once I remembered to set gather_facts to false. Should be fixed now.